src/__tests__/components/Login/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+import LoginModalContent from "@/components/Login";
+import { render, screen } from "@testing-library/react";
+
+describe("Login", () => {
+  test("Each elements in the Login display the correct text", () => {
+    render(<LoginModalContent isOpen={true} onClose={() => void {}} />);
```

Comment:
You can use jest mock functions, check this [link ](https://jestjs.io/docs/jest-object#jestfnimplementation)

Reply:
Tests have been coded without using jest.fn() as per discussed.